app/javascript/schools/settings/components/SchoolCustomize__Root.re

```diff
@@ -24,10 +29,9 @@ type action =
   | UpdatePrivacyPolicy(string)
   | UpdateAddress(string)
   | UpdateEmailAddress(string)
+  | UpdateSchoolDetails(string, option(string))
```

Comment:
When writing dupe types in the same place, consider naming them...

```reason
type action =
  // ...
  | UpdateSchoolDetails(name, about)
and name = string
and about = option(string);
```

...just to improve readability.